### ISSUE TITLE
WIT.ai giving random responses during short periods of the day

### ISSUE THREAD

**Issue by Rolfjdra** [wit-ai/wit#703]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
bug

**What is the current behavior?**
The app gives out random answers(from story), even though it clearly recognizes the input correctly in the understanding tab. 
This usually start happening in the afternoon. It works fine before 1pm.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
When I test the app in the built in chat on WIT.ai it gives out the correct response for the very first input, but then starts to give our random responses again. I have to close the chat window and open it again for it to provide the correct response again.

**What is the expected behavior?**
For the app to provide the response connected to the entity / entity-value

**What is the App ID where you are experiencing this issue (if applicable)?**
5975e26e-10a1-4138-ada3-dab53b2f624c
https://wit.ai/Rolfjdra_066PP/GupshupBot

**Comment by patapizza** [comment]
Hi @Rolfjdra,

Can you show an example of a conversation with a random answer and what is expected instead?

**Comment by Rolfjdra** [comment]
Hi @patapizza 
The behaviour is back to normal again this morning, but I suspect it will happen again in the afternoon. Maybe it is because I do a lot of training in the morning.


**Example of conversation:**
User: "Takk"
Bot: "Ha en fortsatt fin dag!" _(Correct response)_
User: "Takk"
Bot: "Til høyre, oppe i Internet explorer er det et tannhjul, trykk på det og velg det nederste alternativet 
       "om internet explorer". Det vil da komme opp en rute som sier hvilken versjon av internet explorer 
        du benytter. Portalen er optimalisert for internet explorer 11."
User: "Takk"
Bot: "Jeg forstod ikke" _(I did not understand response. Connected to no entity)_
User: "Takk"
Bot:"Til høyre, oppe i Internet explorer er det et tannhjul, trykk på det og velg det nederste alternativet 
       "om internet explorer". Det vil da komme opp en rute som sier hvilken versjon av internet explorer 
       du benytter. Portalen er optimalisert for internet explorer 11."


For the very first input the user enters, it answers correctly, and then it turns to a completely different response, even though it never recognizes the entity connected to this response. It happens for several different inputs aswell.

This only starts happening in the afternoon, and it gives the correct response ("Ha en forstsatt fin dag!") every single time before that. (It is working fine at the moment).

Thank you :)

**Comment by Rolfjdra** [comment]
Today the issues started at 2pm CEST. 

It has chosen a different response to default to this time, but the issue is the same.
It does not misunderstand anything when I enter it in the "Understanding"-field, but when typed into the WIT.ai chat window, it defaults to one of my responses from a different story.

It manages to give the correct response if I close the WIT.ai chat window and reopen it, but only for the first input. Very strange.

**Comment by Rolfjdra** [comment]
It defaults to a new response from a story every morning, and after an hour it is back to normal. I do random changes and resave stories and suddenly it starts working.

Tried to stop using the bot-response in stories and use a function to send the bot-response from my code instead, but it does not make it better. Still defaults to a random story-response.

Will probably be back to normal in an hour, but it is happening every day.

**Comment by patapizza** [comment]
@Rolfjdra What is the time range (with timezone) when it starts to answer randomly?

It seems like it's working now:

![screenshot](http://i.imgur.com/HMwsQkE.png)

**Comment by Rolfjdra** [comment]
@patapizza It has started to happen randomly. I am training it a lot now, so it is hard to say exactly when, but sometimes it happens after I've created a new story / entity.

When it happens, I fix it by changing the title of different stories and re-saving them. This usually fixes the issue, but it is a bit frustrating having this happen on a regular basis (several times a day). 

Maybe it will be more consistent once I have finished the majority of the training.
